README.md

```diff
@@ -76,160 +75,48 @@ points & bars
-forces them to focus on learning APIs rather than exploring their data.
-
-Another challenge is current plotting APIs require the user to write code,
-even for incidental details of a visualization. This results in an unfortunate
-and unnecessary cognitive burden as the visualization type (histogram,
-scatterplot, etc.) can often be inferred using basic information such as the
-columns of interest and the data types of those columns.
-
-For example, if you are interested in the visualization of two numerical
-columns, a scatterplot is almost certainly a good starting point. If you add
-a categorical column to that, you probably want to encode that column using
-colors or facets. If inferring the visualization proves difficult at times, a
-simple user interface can construct a visualization without any coding.
-[Tableau](https://www.tableau.com/) and the [Interactive Data
-Lab's](https://idl.cs.washington.edu/)
-[Polestar](https://github.com/vega/polestar) and
-[Voyager](https://github.com/vega/voyager) are excellent examples of such UIs.
-
-### Design approach and solution
-
-We believe that these challenges can be addressed without the creation of yet
-another visualization library that has a programmatic API and built-in
-rendering. Vega-Altair's approach to building visualizations uses a layered design
-that leverages the full capabilities of existing visualization libraries:
-
-1. Create a constrained, simple Python API (Vega-Altair) that is purely declarative
-2. Use the API (Vega-Altair) to emit JSON output that follows the Vega-Lite spec
-3. Render that spec using existing visualization libraries
-
-This approach enables users to perform exploratory visualizations with a much
-simpler API initially, pick an appropriate renderer for their usage case, and
-then leverage the full capabilities of that renderer for more advanced plot
-customization.
-
-We realize that a declarative API will necessarily be limited compared to the
-full programmatic APIs of Matplotlib, Bokeh, etc. That is a deliberate design
-choice we feel is needed to simplify the user experience of exploratory
-visualization.
+If you are using the conda package manager, the equivalent is:
+```bash
+conda install altair -c conda-forge
+```
 
-## Development install
+For full installation instructions, please see [the documentation](https://altair-viz.github.io/getting_started/installation.html).
 
-Instructions on how to install Vega-Altair for development can be found in
-[the documentation](https://altair-viz.github.io/getting_started/installation.html).
+## Getting Help
+If you have a question that is not addressed in the documentation, 
+you can post it on [StackOverflow](https://stackoverflow.com/questions/tagged/altair) using the `altair` tag
+or ask on the [Vega-Altair Google Group](https://groups.google.com/forum/#!forum/altair-viz).
+For bugs and feature requests, please open a [Github Issue](https://github.com/altair-viz/altair/issues).
```

Comment:
I think we can just link the new doc help page here so that we don't need to keep this info up to date in two different locations. I also don't think we need to mention the google group.

Reply:
I agree, it would be great to link to the new getting help page but do you think that's still ok even though we might not be able to publish it for quite a while? I removed the reference to the google group.

Reply:
Good point, that's something we can save for when the new doc pages are published since the location won't be the same as the old help page.